src/content/upgrades/beacon-chain/index.md

```diff
@@ -6,59 +6,77 @@ template: upgrade
+
+## Beacon Chain impact {#beacon-chain-features}
 
 ### Introducing staking {#introducing-staking}
 
-The Beacon Chain will introduce [proof-of-stake](/developers/docs/consensus-mechanisms/pos/) to Ethereum. This is a new way for you to help keep Ethereum secure. Think of it like a public good that will make Ethereum healthier and earn you more ETH in the process. In practice, it will involve you staking ETH in order to activate validator software. As a validator you'll process transactions and create new blocks in the chain.
+The Beacon Chain introduced [proof-of-stake](/developers/docs/consensus-mechanisms/pos/) to Ethereum. This is a new way for you to help keep Ethereum secure. Think of it like a public good that will make Ethereum healthier and earn you more ETH in the process. In practice, staking involves you staking ETH in order to activate validator software. As a staker you'll run node software that processes transactions and creates new blocks in the chain.
+
+Staking serves a similar purpose to [mining](/developers/docs/mining/), but is different in many ways. Mining requires large up-front expenditures in the form of powerful hardware and energy consumption resulting in economies-of-scale, which promotes centralization. Mining also does not come with any requirement to lock up assets as collateral, limiting the protocols ability to punish bad actors after an attack.
 
-Staking and becoming a validator is easier than [mining](/developers/docs/mining/) (how the network is currently secured). And it's hoped this will help make Ethereum more secure in the long run. The more people that participate in the network, the more decentralized and safe from attack it will become.
+The transition to proof-of-stake will make Ethereum significantly more secure and decentralized by comparison. The more people that participate in the network, the more decentralized and safe from attacks it becomes.
 
 <InfoBanner emoji=":money_bag:">
-If you're interested in becoming a validator and helping secure the Beacon Chain, <a href="/staking/">learn more about staking</a>.
+  If you're interested in becoming a validator and helping secure the Ethereum, <a href="/staking/">learn more about staking</a>.
 </InfoBanner>
 
-This is also an important change for another upgrade: [shard chains](/upgrades/shard-chains/).
+### The Merge and the end of mining {#the-merge}
+
+While the Beacon Chain (or "consensus layer") is already live, it has existed as a separate chain from Mainnet (or the "execution layer") since its genesis. The plan is to swap out the proof-of-work algorithm being used to back the execution layer today, and replace it with the proof-of-stake consensus protocol the Beacon Chain provides.
+
+This process is known as **The Merge**, as it will 'merge' this new consensus layer with the existing execution layer, while deprecating the use of mining.
```

Comment:
```suggestion
This process is known as **The Merge**, as it will 'merge' the new consensus layer with the existing execution layer and stop the use of mining.
```